includes/AJAX.php

```diff
@@ -41,6 +46,55 @@ public function __construct() {
 
 		// get the current sync status
 		add_action( 'wp_ajax_wc_facebook_get_sync_status', [ $this, 'get_sync_status' ] );
+
+		// search a product's attributes for the given term
+		add_action( 'wp_ajax_' . self::ACTION_SEARCH_PRODUCT_ATTRIBUTES, [ $this, 'admin_search_product_attributes' ] );
+	}
+
+
+	/**
+	 * Searches a product's attributes for the given term.
+	 *
+	 * @internal
+	 *
+	 * @since 2.1.0-dev.1
+	 */
+	public function admin_search_product_attributes() {
+
+		try {
+
+			if ( ! wp_verify_nonce( Framework\SV_WC_Helper::get_requested_value( 'security' ), self::ACTION_SEARCH_PRODUCT_ATTRIBUTES ) ) {
+				throw new Framework\SV_WC_Plugin_Exception( 'Invalid nonce' );
+			}
+
+			$term = Framework\SV_WC_Helper::get_requested_value( 'term' );
+
+			if ( ! $term ) {
+				throw new Framework\SV_WC_Plugin_Exception( 'A search term is required' );
+			}
+
+			$product = wc_get_product( (int) Framework\SV_WC_Helper::get_requested_value( 'request_data' ) );
+
+			if ( ! $product instanceof \WC_Product ) {
+				throw new Framework\SV_WC_Plugin_Exception( 'A valid product ID is required' );
+			}
+
+			/** @var array $attributes */
```

Comment:
```suggestion
```
isn't this redundant? the phptag of the method used here already has `@return array` in the phpdoc